.github/workflows/preview-storybook.yml

```
@@ -0,0 +1,87 @@
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          REPO: ${{ github.repository }}
+          STORYBOOK_URL: https://ns-ec-dms-bs3-nimbus-preview.s3.us-west-2.amazonaws.com/components/pull/${{ github.event.pull_request.number }}/index.html
+        run: |
+          echo "Checking for existing Storybook comment..."
+
+          comments=$(curl -s -H "Authorization: token $GH_TOKEN" \
+            https://api.github.com/repos/$REPO/issues/$PR_NUMBER/comments)
+
+          comment_id=$(echo "$comments" | jq -r '.[] | select(.body | test("View Storybook.*pull/'"$PR_NUMBER"'")) | .id')
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)